spec/requests/clients_spec.rb

```diff
@@ -1,77 +1,50 @@
 require 'rails_helper'
 
-RSpec.describe 'Clients', type: :request, elasticsearch: true  do
-  let!(:provider) { build(:provider) }
```

Comment:
This is the difference when using FactoryBot without ActiveRecord. You can not use *create factory*. You need to build them and then create them as a secondary step.